azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -892,10 +892,27 @@ private void insertJVMAargs() {
     final String previousJVMArgs = this.props.get(JavaProcessJob.JVM_PARAMS);
     jobJVMArgs += (previousJVMArgs == null) ? "" : " " + previousJVMArgs;
 
+    // Add useful Java options for java jobs which are provided through properties
+    String javaOpts = insertJavaOptions();
+    jobJVMArgs += (javaOpts == null) ? "" : " " + javaOpts;
     this.logger.info("job JVM args: " + jobJVMArgs);
     this.props.put(JavaProcessJob.JVM_PARAMS, jobJVMArgs);
   }
 
+  /**
+   * Add useful Java options for java jobs provided through properties.
+   */
+  private String insertJavaOptions() {
+    if (this.jobtypeManager.getCommonPluginLoadProps() == null) {
```

Comment:
Which property file is expected to contain the new prop values ?

Reply:
It is in commonprivate.properties